docs/source/index.rst

```diff
@@ -55,83 +55,94 @@ processo de submissão e comunidade de usuários.
 Ferramentas de apoio
 --------------------
 
-Algumas ferramentas são disponibilizadas e mantidas pelo SciELO, seguindo o modelo 
-:term:`open-source`, para apoiar o processo de marcação dos documentos no 
-formato XML. 
-
-* `Markup <http://docs.scielo.org/projects/scielo-pc-programs/en/latest/markup.html>`_: 
-  Macro para Microsoft Word que apoia o processo de marcação de documentos, 
-  de acordo com a `DTD PMC <http://dtd.nlm.nih.gov/publishing/3.0/>`_.
-* `Stylechecker <http://manager.scielo.org/tools/validators/stylechecker/>`_: 
-  Ferramenta baseada na web que apresenta relatório detalhado sobre a 
-  conformidade de um dado XML em relação à especificação :term:`SciELO PS`.
-* `Packtools <https://github.com/scieloorg/packtools/>`_: Biblioteca 
-  :term:`Python` que agrega funcionalidades e utilitários para a manipulação 
-  de :term:`pacotes SciELO PS` e XMLs :term:`SciELO PS`.
+Algumas ferramentas são disponibilizadas e mantidas pela SciELO seguindo o
+modelo :term:`open-source`, para apoiar o processo de marcação de documentos em
+formato XML.
```

Comment:
OK, vejamos então o caso de um texto acadêmico. Quando surge a primeira vez um termo ou autor, este é identificado e faz-se uma referência (nota de rodapé, nota geral, citação ou indicação de onde há um esclarecimento sobre o texto). Todas as demais ocorrências naquela página e seguintes, encontram-se em texto normal. Assume-se que a pessoa já leu. Creio que marcar uma vez por arquivo seja suficiente e somente no início, mas se vocês preferem que seja marcado sempre, pra mim não há problema.